go.mod

```diff
@@ -60,3 +60,5 @@ require (
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
 	gopkg.in/yaml.v2 v2.2.8 // indirect
 )
+
+replace github.com/pingcap/kvproto => github.com/rleungx/kvproto v0.0.0-20200305052806-a32ceefc091e
```

Comment:
Where is your PR about this?

Reply:
https://github.com/rleungx/kvproto/tree/sync-statistic